src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -2802,6 +2802,7 @@ static void Main(string[] args)
         <entry offset=""0x1b"" startLine=""13"" startColumn=""13"" endLine=""13"" endColumn=""14"" document=""1"" />
         <entry offset=""0x1c"" startLine=""14"" startColumn=""17"" endLine=""14"" endColumn=""33"" document=""1"" />
         <entry offset=""0x23"" hidden=""true"" document=""1"" />
+        <entry offset=""0x25"" hidden=""true"" document=""1"" />
```

Comment:
> <entry offset=""0x25"" hidden=""true"" document=""1"" /> [](start = 7, length = 57)

Can we avoid emitting an extra hidden sequence point in non-pattern-matching switch statements?

Reply:
> Can we avoid emitting an extra hidden sequence point in non-pattern-matching switch statements?

There is no such thing any longer. All `switch` statements are pattern-matching switch statements.

Reply:
Sure, but we don't need this hidden sequence point for the trivial case clauses, do we?